src/routes/login.tsx

```
@@ -1,109 +1,44 @@
-import React, { useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
-import { auth } from "../firebase";
+import { BasicInput } from "../components/ui/Input";
+import ExternalLogin from "../components/auth/ExternalLogin";
+import { useLogin } from "../hooks/auth";
 import {
-  Wrapper,
-  Title,
-  Form,
-  Input,
   Error,
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)